projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.scss

```diff
@@ -0,0 +1,38 @@
+.commentSummaryHeader {
+  width: 24%;
```

Comment:
I assume this changes based on whether the comments panel is shown or hidden

Reply:
it is all with the scss just below is code that manages the shown or hidden